CHANGELOG.mdx

```diff
@@ -17,105 +17,121 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 <!-- TOWNCRIER -->
 
 ## [3.6.8] - 2023-08-30
-                       
-Rasa 3.6.8 (2023-08-30)                         
 
-No significant changes.
+Rasa 3.6.8 (2023-08-30)
```

Comment:
do you know what happened here, why is the diff so large?

Reply:
I think it's just my editor that removed some trailing whitespaces. If you look at the lines below, the actual content is not affected